content/attacks/posts/2019-03-24-DragonEx.md

```diff
@@ -0,0 +1,34 @@
+---
+date: 2019-03-24
+categories: "Custodian"
+title: "DragonEx Hacked for $7 Million"
+---
+
+## Summary
+
+On March 24, 2019, [DragonEx, a Singapore-based cryptocurrency exchange, was hacked for $7 million](https://www.coindesk.com/markets/2019/03/26/singapore-based-crypto-exchange-dragonex-has-been-hacked/). The attacker gained access to one of the exchange's hot wallets, which are used to store user funds that are available for withdrawal.
+
+## Attackers
+
+Attackers associated with [Lazarus Group](https://bitcoinmagazine.com/culture/lazarus-hacker-group-continues-target-crypto-using-faked-trading-software) gained access through a sophisticated phishing attack. [More about Lazarus](https://go.chainalysis.com/rs/503-FAP-074/images/2020-Crypto-Crime-Report.pdf)
+
+ETH 
+- DragonEx Hacker wallet: [0xa7f72Bf63EDeCa25636F0B13Ec5135296ca2eBb2](https://etherscan.io/address/0xa7f72Bf63EDeCa25636F0B13Ec5135296ca2eBb2)
+
+## Losses
+
+The attackers managed to steal a total of $7 million:
+- 4.3 million USDT
+- 2.2 million in BNB
+- 300,000 USDC 
+
+## Timeline
+
+- **March 24, 2022:** The attacker gained access to one of the exchange's hot wallets and began withdrawing funds.
+- **March 25, 2022:** [DragonEx became aware of the hack and suspended withdrawals.](https://t.me/DragonEx_EN/50126)
+- **March 26, 2022:** DragonEx announced that the hack had resulted in a loss of $7 million.
```

Comment:
Provide source please

Reply:
it often happens that different articles link to a deleted post. In this case, do not write about it?

Reply:
I can't find information about the dates of their announcement, but I was able to find an articles from other sources which provides screenshots to posts/messages published on 25th March. So, it would be better to delete the line, since previous line refers to the same